src/sentry/search/eap/span_columns.py

```diff
@@ -386,6 +402,65 @@ def count_processor(count_value: int | None) -> int:
         return count_value
 
 
+def http_response_rate(arg: str) -> Column.BinaryFormula:
+
+    if not arg.isdigit():
```

Comment:
nit: why not just check that arg is in a list of the strings of 1-5, then you can cast safely without concern?